public/app/features/alerting/unified/components/contact-points/ContactPoints.tsx

```diff
@@ -58,16 +69,32 @@ const ContactPointsTab = () => {
 
   const search = queryParams.get('search');
 
-  if (error) {
-    // TODO fix this type casting, when error comes from "getContactPointsStatus" it probably won't be a SerializedError
-    return <Alert title="Failed to fetch contact points">{stringifyErrorLike(error)}</Alert>;
-  }
-
   if (isLoading) {
     return <LoadingPlaceholder text="Loading..." />;
   }
 
   const isGrafanaManagedAlertmanager = selectedAlertmanager === GRAFANA_RULES_SOURCE_NAME;
+
+  if (contactPoints.length === 0) {
```

Comment:
I don't think it was possibly/very likely at all to see an empty state before, as there was a default contact point, and it was tricky to end up deleting everything